### ISSUE TITLE
where does 'get this as a chrome ext.' link from landing go?

### ISSUE THREAD

**Issue by heatherbooker** [heatherbooker/graphmarks#16]
can we just make it open the download/installer?

**Comment by heatherbooker** [comment]
so we can make it open graphmarks in the chrome store, or directly install from one click - i like the first option; [more info here](https://developer.chrome.com/webstore/inline_installation#cws-link).
it would need to be a link like
```html
a(href="https://chrome.google.com/webstore/detail/itemID") Click to Install Chrome Extension
```
and the itemID would be ... hold on, I think maybe we can just copy and paste the URL when we open the extension installation page ourselves. derp.

**Comment by ebb-tide** [comment]
agreed! the first option feels way less intrusive. will it look like this?
![image](https://cloud.githubusercontent.com/assets/10937678/23346094/2ec54900-fc65-11e6-9e32-36e923c79ba4.png)

**Comment by heatherbooker** [comment]
yes!
